obelisk_ws/src/obelisk_ros/config/leap_cpp.yaml

```diff
@@ -0,0 +1,113 @@
+          # key: sub1
+          topic: /obelisk/leap/est
+          msg_type: EstimatedState
+          history_depth: 10
+          # callback_key: sub_callback1
+          callback_group: None
+          non_obelisk: False
+      timers:
+        - ros_parameter: timer_ctrl_setting
+          # key: timer1
+          timer_period_sec: 0.001
+          callback_group: None
+          # callback_key: timer_callback1
+  estimation:
+    - pkg: obelisk_estimation_cpp
+      executable: jointencoders_passthrough_estimator
+      # callback_groups:
+      publishers:
+        - ros_parameter: pub_est_setting
+          # key: pub1
+          topic: /obelisk/leap/est
+          msg_type: EstimatedState
+          history_depth: 10
+          callback_group: None
+          non_obelisk: False
+      subscribers:
+        - ros_parameter: sub_sensor_setting
+          # key: sub1
+          topic: /obelisk/leap/sensor
+          msg_type: JointEncoders
+          history_depth: 10
+          # callback_key: sub_callback1
+          callback_group: None
+          non_obelisk: False
+      timers:
+        - ros_parameter: timer_est_setting
+          # key: timer1
+          timer_period_sec: 0.001
+          callback_group: None
+          # callback_key: timer_callback1
+  # sensing:
+  robot:
+    # === hardware ===
+    # - is_simulated: False
+    #   pkg: obelisk_leap_cpp
+    #   executable: obelisk_leap_robot
+    # === simulation ===
+    - is_simulated: True
+      pkg: obelisk_sim_cpp
+      executable: obelisk_mujoco_robot
+    # =================
+      # callback_groups:
+      publishers:
+        - ros_parameter: pub_sensor_setting
+          topic: /obelisk/leap/sensor
+          msg_type: JointEncoders
+          history_depth: 10
+          callback_group: None
+          non_obelisk: False
+      subscribers:
+        - ros_parameter: sub_ctrl_setting
+          topic: /obelisk/leap/ctrl
+          msg_type: PositionSetpoint
+          history_depth: 10
+          callback_group: None
+          non_obelisk: False
+      sim:
+        - ros_parameter: mujoco_setting
+          model_xml_path: leap_rh/leap_palm_up.xml
+          n_u: 16
+          time_step: 0.002
+          num_steps_per_viz: 5
+          sensor_settings:
+          - topic: /obelisk/leap/sensor
+            dt: 0.002
+            sensor_type: jointpos
+            sensor_names:
+            - if_mcp_sensor
+            - if_rot_sensor
+            - if_pip_sensor
+            - if_dip_sensor
+            - mf_mcp_sensor
+            - mf_rot_sensor
+            - mf_pip_sensor
+            - mf_dip_sensor
+            - rf_mcp_sensor
+            - rf_rot_sensor
+            - rf_pip_sensor
+            - rf_dip_sensor
+            - th_cmc_sensor
+            - th_axl_sensor
+            - th_mcp_sensor
+            - th_ipl_sensor
+      timers:
+        - ros_parameter: timer_sensor_setting
+          timer_period_sec: 0.02
+          callback_group: None
```

Comment:
delete this unnecessary timer. Also delete it on `leap_py.yaml`, I just noticed it.

Reply:
I need a timer to read and publish the state of the robot.

Reply:
resolved in slack.